scripts/db_migrator.py

```diff
@@ -1000,12 +1000,14 @@ def version_4_0_0(self):
         # reading FAST_REBOOT table can't be done with stateDB.get as it uses hget behind the scenes and the table structure is
         # not using hash and won't work.
         # FAST_REBOOT table exists only if fast-reboot was triggered.
-        keys = self.stateDB.keys(self.stateDB.STATE_DB, "FAST_REBOOT|system")
-        if keys:
-            enable_state = 'true'
-        else:
-            enable_state = 'false'
-        self.stateDB.set(self.stateDB.STATE_DB, 'FAST_RESTART_ENABLE_TABLE|system', 'enable', enable_state)
+        keys = self.stateDB.keys(self.stateDB.STATE_DB, "FAST_RESTART_ENABLE_TABLE|system")
+        if not keys:
+            keys = self.stateDB.keys(self.stateDB.STATE_DB, "FAST_REBOOT|system")
```

Comment:
@arfeigin how do we address the conversion case where we want to set the flag, DB will not have it?

I remember we had a discussion about this in the past w/ your fastboot over warmboot infra changes. But, can't fully recollect the conclusion.

Reply:
I think this is the discussion we had then: https://github.com/sonic-net/sonic-utilities/pull/2621#discussion_r1130519315.
I think the solution was to have the entry FAST_REBOOT_ENABLE_TABLE|system {enable, <true/false>} according to the state.
In the suggested implementation as I see:

1. If entry exists it will be preserved with either true/false.
2. If entry doesn't exist, if it had old fast-reboot entry set (fast-reboot from older version) it will be added with enable_state=true otherwise it will be added with enable_state=false.
So as I see in any case the entry will be present in state-db, either with previous values or it will be added.
In both cases it will hold accurate data as I understand.

Reply:
Hi @vaibhavhd , could you please check @arfeigin 's reply?